### PR TITLE
DOC: Add missing parameters to offsets class: Tick

### PR DESCRIPTION
**Pull request by Anh-HoDac**
- [x] partially complete #52431
Updated documentation for offsets class: Tick (added missing parameters).

### CODE REVIEW

File: pandas/_libs/tslibs/offsets.pyx [hunk 1]
```diff
@@ -934,6 +934,20 @@ cdef class SingleConstructorOffset(BaseOffset):
...
+    ----------
+    n : int, default 1
+        The number of ticks represented.
+    normalize : bool, default False
+        Normalize start/end dates to midnight before generating date range.
+
+    See Also
+    --------
+    :class:`~pandas.tseries.offsets.DateOffset` : Standard kind of date increment.
+
```
Comment (Aloqeely) [inline pandas/_libs/tslibs/offsets.pyx:950]:
```suggestion
    """
```
(You forgot to close the docstring)

**Review by Aloqeely — COMMENTED** [review]
Thanks for the PR!

**Comment by Anh-HoDac** [comment]
Thank you @Aloqeely! I have committed to the suggested change.

**Comment by mroeschke** [comment]
Could you address these failing check? 

```
Error: None:None:EX01:pandas.tseries.offsets.Tick:No examples section found
Error: None:None:GL08:pandas.tseries.offsets.Tick:EXPECTED TO FAIL, BUT NOT FAILING
```

**Comment by MarcoGorelli** [comment]
hi @Anh-HoDac - this looks to be on the right track, if you can add an example to the docstring I think it can go in

**Comment by mroeschke** [comment]
Thanks for the pull request, but it appears to have gone stale. If interested in continuing, please merge in the main branch, address any review comments and/or failing tests, and we can reopen.

**Pull request closed without merging**